packages/frameworks/test/frameworks.unit.test.ts

```diff
@@ -97,6 +97,23 @@ const Schema = {
           outputDirectory: SchemaSettings,
         },
       },
+      recommendedIntegrations: {
+        type: 'array',
```

Comment:
It needs `required: [id, dependencies]` and `additionalProperties: false` right?

Reply:
@styfle indeed 👍